.go-env.sh

```diff
@@ -4,12 +4,25 @@ set -Eeuo pipefail
 dir="$(dirname "$BASH_SOURCE")"
 dir="$(readlink -ve "$dir")"
 
-user="$(id -u):$(id -g)"
+src="$dir"
+dst="$dir"
+msys=
+if [ "$(uname -o)" = 'Msys' ];	then
+	msys=1
+	if command -v cygpath > /dev/null; then
+		src="$(cygpath.exe --windows "$dst")"
+	fi
+fi
+windowsContainers=
+if [ "$(docker version --format '{{ .Server.Os }}')" = 'windows' ]; then
+	windowsContainers=1
+	dst='c:\app'
+fi
+
 args=(
 	--interactive --rm --init
-	--user "$user"
-	--mount "type=bind,src=$dir,dst=$dir"
-	--workdir "$dir"
+	--mount "type=bind,src=$src,dst=$dst"
+	--workdir "$dst"
 	--tmpfs /tmp,exec
 	--env HOME=/tmp
```

Comment:
I'm honestly (pleasantly) surprised this ~works on Windows!  The paths are screwy, but that it works is excellent. :+1: